airgun/views/login.py

```diff
@@ -4,7 +4,7 @@
 class LoginView(View, ClickableMixin):
     username = TextInput(id='login_login')
     password = TextInput(id='login_password')
-    submit = Text('//input[@name="commit"]')
+    submit = Text('//button[@type="submit"]')
```

Comment:
So far downstream is not impacted around it as this issue exists on Foreman v1.23/24 ... Hence please add the label 'Do Not Merge' on it until comes under downstream part.

Reply:
No probs.. lets just keep it open until it becomes relevant

Reply:
Please add label 'DO NOT MERGE' on this PR so that it will not merge until you requires .

Reply:
I have added Do Not Merge label, once we see that test requires. we can merge this.